conda

```diff
@@ -5,18 +5,36 @@ tags: [ packaging ]
 conda env list
 
 # To initialise an environment
-conda create --name 'ENVNAME'
+conda create --name <environment_name>
 
-# To initialise an environment with python 
-conda create --name 'ENVNAME' python=3.10
+# To initialise an environment with python3.10 
+conda create --name <environment_name> python=3.10
 
-# To install from requirements.txt
-conda install --file requirements.txt
+# To install from a file
+conda install --file <requirements.txt>
+
+# To clone an environment
+conda create --clone <old_environment_name> --name <new_environment_name>
 
 # To activate a virtual environment
-conda activate 'ENVNAME'
+conda activate <environment_name>
+
+# To deactivate a virtual environment
+conda deactivate
 
 # To remove and environment
```

Comment:
Please change "To remove and environment" to "To remove an environment" to fix a spelling error.

Reply:
done, sorry for missing that!